packages/client/machines/creationMpeRoomForm.ts

```diff
@@ -0,0 +1,280 @@
+                                {
+                                    isOpen: false,
+                                },
+                            ),
+                        },
+                    },
+
+                    on: {
+                        SET_OPENING_STATUS: [
+                            {
+                                cond: (_context, { isOpen }) => isOpen === true,
+
+                                target: '.public',
+                            },
+
+                            {
+                                target: '.private',
+                            },
+                        ],
+
+                        NEXT: {
+                            target: 'confirmation',
+                        },
+
+                        GO_BACK: {
+                            target: 'roomName',
+                        },
+                    },
+                },
+
+                confirmation: {
+                    entry: creationMpeRoomFormModel.actions.redirectToScreen({
+                        screen: 'MusicPlaylistEditorCreationFormConfirmation',
+                    }),
+
+                    initial: 'fetchingInitialTracksInformation',
+
+                    states: {
+                        fetchingInitialTracksInformation: {
+                            invoke: {
+                                src: 'fetchTracksInformation',
+                            },
+
+                            on: {
+                                RECEIVED_INITIAL_TRACKS_METADATA: {
+                                    target: 'debouncing',
+
+                                    actions:
+                                        assignInitialTracksMetadataToContext,
+                                },
+                            },
+                        },
+
+                        debouncing: {
+                            after: {
+                                1000: {
+                                    target: 'fetchedInitialTracksInformation',
+                                },
+                            },
+                        },
+
+                        fetchedInitialTracksInformation: {
+                            tags: 'hasFetchedInitialTracksInformation',
+                        },
+                    },
+
+                    on: {
+                        GO_BACK: {
+                            target: 'openingStatus',
+                        },
+
+                        NEXT: {
+                            target: 'confirmed',
+                        },
+                    },
+                },
+
+                confirmed: {
+                    type: 'final',
+
+                    data: ({ roomName, isOpen, onlyInvitedUsersCanVote }) => ({
+                        roomName,
+                        isOpen,
+                        onlyInvitedUsersCanVote,
+                    }),
+                },
+            },
+        },
+        {
+            actions: {
```

Comment:
interesting xstate model usage, finally the global scoped actions
Typescript linter looks like not ready for this haha